poetry.lock

```diff
@@ -845,27 +845,28 @@ files = [
 
 [[package]]
 name = "deepeval"
-version = "2.3.2"
+version = "2.5.1"
 description = "The Open-Source LLM Evaluation Framework."
 optional = false
 python-versions = "<3.14,>=3.9"
 groups = ["test"]
 files = [
-    {file = "deepeval-2.3.2-py3-none-any.whl", hash = "sha256:e32e39c5c1c7606a4850169bd4c0ff72809eabcc30eed91b4d6ee451491a3f3c"},
-    {file = "deepeval-2.3.2.tar.gz", hash = "sha256:8f52768908e5cfbfb3e9ca772d0605f98c2500ae874f9657bc43df36227a4a41"},
+    {file = "deepeval-2.5.1-py3-none-any.whl", hash = "sha256:66e2c65a0596593dd4818c307a280f52b6e710dff7283194a2d31363e4806107"},
+    {file = "deepeval-2.5.1.tar.gz", hash = "sha256:f4361b00ca94b7a1d7e37c51391882b6c487912772c7b17683cd448f7795456a"},
 ]
 
 [package.dependencies]
 datasets = "*"
 docx2txt = ">=0.8,<1.0"
-grpcio = "1.67.1"
+grpcio = ">=1.67.1,<2.0.0"
 importlib-metadata = ">=6.0.2"
 langchain = "*"
 langchain-community = "*"
 langchain-core = "*"
 langchain_openai = "*"
 llama-index = "*"
 nest-asyncio = "*"
+ollama = "*"
```

Comment:
why do we need ollama?

Reply:
Because we update deepeval 2.3.2 to 2.5.1 and it added support for ollama. These are the indirect dependencies introduced by our direct dependencies.